app/Resources/views/page.html.twig

```diff
@@ -133,7 +133,10 @@
             <meta property="og:image:width" content="1114">
             <meta property="og:image:height" content="336">
         {% else %}
-            <meta name="twitter:card" content="summary">
+            <meta name="twitter:card" content="summary_large_image">
+            <meta property="og:image" content="{{ absolute_url(asset('assets/images/social/icon-1200x630@1.png')) }}">
```

Comment:
I believe @chugginselifesciences has said for all images to be 600px x 600px as a starting point from the conversation I had with him this morning.